src/css/math.less

```diff
@@ -298,9 +298,27 @@
     }
   }
 
+  .mq-overarrow {
+    border-top: 1px solid black;
+    margin-top: 1px;
+    padding-top: 0.2em;
 
-  &, .mq-editable-field {
-    cursor: text;
-    font-family: @symbola;
```

Comment:
This appears to remove Symbola from all of `.mq-math-mode` (that's what the `&` refers to in this context)